bcmserver.c

```diff
@@ -197,7 +197,11 @@ int main(void)
 	while (1) { 
 		sa = accept(sl,(struct sockaddr *)&clientaddr, &sin_size);
 		if (sa > 0 ){
+#ifdef HAVE_FORK
 			if (!fork())
+#else
+			if (!vfork())
+#endif
```

Comment:
I would prefer to have a can utils internal helper function to avoid of spreading of ifdefs over the code. Probably in lib.c with the name some_prefix_fork. It looks like we do not have internal naming schema to avoid confusions.
How about canutils_fork or cu_fork? The last variant seems to be short enough and still some how reflect relation to can utils. Folks, what do you think?